docs/BasicCommunication/PolicyUpdate/index.md

```diff
@@ -12,53 +12,64 @@ Purpose
 
 ``BC.PolicyUpdate`` represents SDL-generated request to start the PTU sequence.
 
-!!! must
+_**HMI must:**_
```

Comment:
All of these changes that you made must be made throughout the entire PR, such as these `!!! MUST`/ `!!! NOTE` issues